cdc/sink/mq/mq_flush_worker_test.go

```diff
@@ -173,11 +172,15 @@ func TestBatch(t *testing.T) {
 		},
 	}
 
-	var wg sync.WaitGroup
 	ctx := context.Background()
-	batch := make([]mqEvent, 3)
-	for _, test := range tests {
+	for i := range tests {
+		test := tests[i]
 		t.Run(test.name, func(t *testing.T) {
+			t.Parallel()
+
+			batch := make([]mqEvent, 3)
```

Comment:
We shouldn't create a new batch here, it turns out to test its correctness when using the same block of memory.